### PR TITLE
Puppet 4 compability

### PR DESCRIPTION
**Pull request by jkroepke**
puppet 4 bin path is /opt/puppetlabs/puppet/bin, not /usr/bin.

### CODE REVIEW

**Comment by jkroepke** [comment]
For some reason, the variables from parseyaml do not have a _ prefix.
On ubuntu, the default pidfile from the agent config will not used.

File: check_puppet_agent [hunk 1]
```diff
@@ -232,8 +235,10 @@ if [ $daemonized -eq 1 ];then
   fi
 fi
 
+[ $puppet_major_version -ge 3 ] && prefix=_
```
Comment (aswen) [inline check_puppet_agent]: Thanks for your PR. Are you sure about this version number? Shouldn't that be 4?

**Comment by aswen** [comment]
Hi @jkroepke : Thanks for your PR. I haven't got any version 4 installs so far to test on so I wonder if you tested the plugin with your changes on both Puppet 3 and 4 machines?

**Comment by jkroepke** [comment]
Hi,
since I upgrade all my nodes to puppet 4, no. Anyway. https://github.com/aswen/nagios-plugins/pull/19 does have a better code quality.

**Comment by aswen** [comment]
Hi @jkroepke : Thanks for your prompt reply. I shall merge #19 then. Please feel free to re-request your idea of having an option to set the pidfile which I liked very much!

**Pull request closed without merging**